[debug] [2024-08-20T16:21:14.876Z] ----------------------------------------------------------------------
[debug] [2024-08-20T16:21:14.881Z] Command:       /Users/moneibakang/.nvm/versions/node/v18.20.3/bin/node /Users/moneibakang/.nvm/versions/node/v18.20.3/bin/firebase functions:log
[debug] [2024-08-20T16:21:14.882Z] CLI Version:   13.15.2
[debug] [2024-08-20T16:21:14.883Z] Platform:      darwin
[debug] [2024-08-20T16:21:14.883Z] Node Version:  v18.20.3
[debug] [2024-08-20T16:21:14.887Z] Time:          Tue Aug 20 2024 18:21:14 GMT+0200 (Central Africa Time)
[debug] [2024-08-20T16:21:14.888Z] ----------------------------------------------------------------------
[debug] 
[debug] [2024-08-20T16:21:15.014Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2024-08-20T16:21:15.015Z] > authorizing via signed-in user (bakangmonei2@gmail.com)
[debug] [2024-08-20T16:21:15.016Z] [iam] checking project receipts-for-tuition for permissions ["firebase.projects.get","logging.logEntries.list","logging.logs.list"]
[debug] [2024-08-20T16:21:15.018Z] > refreshing access token with scopes: []
[debug] [2024-08-20T16:21:15.020Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-20T16:21:15.021Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-20T16:21:25.394Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------051611363330988163179669","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-20T16:21:25.395Z] [iam] error while checking permissions, command may fail: FirebaseError: Authentication Error: Your credentials are no longer valid. Please run firebase login --reauth

For CI servers and headless environments, generate a new token with firebase login:ci
[debug] [2024-08-20T16:21:25.397Z] > refreshing access token with scopes: []
[debug] [2024-08-20T16:21:25.398Z] >>> [apiv2][query] POST https://www.googleapis.com/oauth2/v3/token [none]
[debug] [2024-08-20T16:21:25.398Z] >>> [apiv2][body] POST https://www.googleapis.com/oauth2/v3/token [omitted]
[debug] [2024-08-20T16:21:35.431Z] *** [apiv2] error from fetch(https://www.googleapis.com/oauth2/v3/token, {"headers":{},"method":"POST","body":{"_overheadLength":545,"_valueLength":212,"_valuesToMeasure":[],"writable":false,"readable":true,"dataSize":0,"maxDataSize":2097152,"pauseStreams":true,"_released":true,"_streams":[],"_currentStream":null,"_insideLoop":false,"_pendingNext":false,"_boundary":"--------------------------091764174321903069388285","_events":{},"_eventsCount":1}}): FetchError: request to https://www.googleapis.com/oauth2/v3/token failed, reason: getaddrinfo ENOTFOUND www.googleapis.com
[debug] [2024-08-20T16:21:40.452Z] Error: Timed out.
    at Timeout._onTimeout (/Users/moneibakang/.nvm/versions/node/v18.20.3/lib/node_modules/firebase-tools/lib/utils.js:268:49)
    at listOnTimeout (node:internal/timers:569:17)
    at process.processTimers (node:internal/timers:512:7)
[error] 
[error] Error: An unexpected error has occurred.
